mirrord-layer/src/lib.rs

```diff
@@ -81,6 +81,14 @@ pub(crate) static ENABLED_UDP_OUTGOING: OnceLock<bool> = OnceLock::new();
 
 #[ctor]
 fn init() {
```

Comment:
rename `init` to `before_init` and rename `init_detour` to `init`.
other names are welcome but `init_detour` is just wrong